src/ethereum/truffle/rad2sol.js

```diff
@@ -31,17 +33,35 @@ const steps = [
   pack,
   (request) => intoProtoBuf(request, schema),
   buff => buff.toString("hex"),
+  (hex, i) => {
+    const priceName = requestNames[i].replace(/\.js/, "")
+    migrationsRadons[`${priceName}Feed`] = {
+      ERC2362ID: 
+        `Price-${priceName.replace("Price", "")
+          .split(/(?=[A-Z])/)
+          .map(s => s.toUpperCase())
+          .join("/")
+        }-3`,
+      bytecode:
+        "0x" + hex
+    }
+    return hex
+  },
   (hex, i) => intoSol(hex, requestNames[i]),
   (sol, i) => writeSol(sol, requestNames[i], requestContractsDir, fs),
 ];
 
 requestsBanner();
 
-Promise.all(steps.reduce(
-  (prev, step) => prev.map((p, i) => p.then(v => step(v, i))),
-  requestNames.map(fileName => Promise.resolve(fileName))))
+Promise.all(
+  steps.reduce(
+    (prev, step) => prev.map((p, i) => p.then(v => step(v, i))), 
+    requestNames.map(fileName => Promise.resolve(fileName))
+  )
+)
   .then(requestsSucceed)
   .then(migrationsBanner)
-  .then(() => writeMigrations(contractNames, userContractsDir, migrationsDir, fs))
+  // .then(() => writeMigrations(contractNames, userContractsDir, migrationsDir, fs))
```

Comment:
When I try this branch while being in `witnet-price-feed-examples`, it is generating `radons.json`, but I'm getting no migration files whatsoever, apparently because this line is commented out.